<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PATS(Piggle Automated Trading System)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pigicon.ico</normaloff>pigicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_manageConditions">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>1111</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_totalBalance">
     <attribute name="title">
      <string>총 잔고</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1091</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="bx_tab_totalBalance">
       <item>
        <widget class="QTableWidget" name="tbl_totalBalance"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="bt_updateAccount">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>154</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>새로고침</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_autoOrder">
     <attribute name="title">
      <string>자동 매매 기록</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>1091</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="bx_tab_autoOrder">
       <item>
        <widget class="QTableWidget" name="tbl_chejan"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_managerAutoOrder">
     <attribute name="title">
      <string>단일 종목 매매</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1091</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="bx_tab_manageAutoOrders">
       <item>
        <widget class="QTableWidget" name="tbl_manageConditions"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="bt_addCondition">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_stopAll_condition">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>전체 중지</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_deleteCondition">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>삭제</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_editCondition">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>수정</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_stop_condition">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>선택 중지</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_start_condition">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>선택 시작</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_kiwoomConditions">
     <attribute name="title">
      <string>키움 조건 매매</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>1091</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="bx_tab_kiwoomConditions">
       <item>
        <widget class="QTableWidget" name="tbl_manageKiwoomConditions"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="bt_editConditionKW">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>수정</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_addConditionKW">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_deleteConditionKW">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>삭제</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_stopAll_conditionKW">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>전체 중지</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_stop_conditionKW">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>선택 중지</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_start_conditionKW">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>선택 시작</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_manageAIConditions">
     <attribute name="title">
      <string>AI 자동매매</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>1091</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="bx_tab_AIConditions"/>
     </widget>
     <widget class="QTableWidget" name="tbl_manageAIConditions">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1089</width>
        <height>129</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_stopAll_conditionAI">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>전체 중지</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_start_conditionAI">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>선택 시작</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_stop_conditionAI">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>선택 중지</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_editAICondition">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>수정</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_addAICondition">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_deleteAICondition">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>삭제</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_piggleDaoMostVoted">
     <attribute name="title">
      <string>DAO 투표값 매매</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1091</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tbl_piggleDaoMostVoted"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="bt_updatePiggleDaoMostVoted">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>154</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>새로고침</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_settingPiggleDaoMostVoted">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>154</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>설정</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>1121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>410</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>자동 주문 로그</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="tv_atLog">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>430</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_stockList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_candleList">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>8</y>
      <width>106</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>캔들</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>77</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>보조지표</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>831</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="bx_chartArea"/>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="bx_subIndexListArea">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QPushButton" name="bt_favorite">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>2</y>
      <width>31</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="tv_favorite">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>300</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>280</y>
      <width>77</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>즐겨찾기</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_realPrice">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>50</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_chartPrev">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>574</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>◀</string>
    </property>
   </widget>
   <widget class="QLabel" name="tv_chartCurrentPage">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>576</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>576</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="tv_chartTotalPage">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>577</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_chartNext">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>574</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
